app/daemon_test.go

```diff
@@ -52,6 +52,10 @@ var (
 	testData              string
 )
 
+func init() {
+	connectionmanager.SetDefaultPingWait(10 * time.Second)
```

Comment:
Since the test is one single binary, this affects all unit tests in this package. Is that intended, or would it be better to add this to setup/teardown functions?

Reply:
It is intended, because the almost all the unit tests expect a lower value for ping timeouts.
This applies to all the unit tests in the `app` package, indeed.